bool    Arm_mode
bool    Motor_EN
int16   Camera_Distance
float32 Joint_1
float32 Joint_2
float32 Joint_3
float32 Joint_4
float32 Joint_5
bool    Gripper